### PR TITLE
3839 split data status

### LINKED ISSUE #3839: Split `status` and `pageData` into their own props in App.jsx
**Is your feature request related to a problem? Please describe.**
Following https://github.com/bbc/simorgh/pull/3780 we need status seperately to app. To get it, we need to split it out in `withContexts` which is a bit gross

**Describe the solution you'd like**
Stop `App.jsx` returning `data`, instead return `statusCode` and `pageData`

**Describe alternatives you've considered**

**Testing notes**
Page status for articles and frontpages shouldn't be affected by this change.General regression required.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by mykeels**
Resolves #3839

**Overall change:** _Split `status` and `pageData` into their own props in App.jsx._

**Code changes:**

- _Changed `data` props in Components to `pageData` and `status`._
- _Modified tests, to meet changes.._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/FrontPage/index.test.jsx [hunk 1]
```diff
@@ -147,8 +147,8 @@ describe('FrontPageContainer', () => {
           <FrontPageComponent
             {...defaultProps}
             error={null}
-            data={data}
             service="igbo"
+            {...data}
```
Comment (dr3) [inline src/app/containers/FrontPage/index.test.jsx]: Can you explicitly split this up on line 140 please?

File: src/app/containers/App/App.jsx [hunk 2]
```diff
@@ -41,7 +44,8 @@ export const App = ({ routes, location, initialData, bbcOrigin, history }) => {
       } = getRouteProps(routes, location.pathname);
 
       setState({
-        data: null,
+        pageData: null,
+        status: 500,
```
Comment (dr3) [inline src/app/containers/App/App.jsx]: This shouldnt be 500, an absense of data during loading isnt an error

Comment (mykeels) [reply src/app/containers/App/App.jsx]: So, 0? Or 404?

Comment (dr3) [reply src/app/containers/App/App.jsx]: Probably null tbh

**Review by FK78 — APPROVED** [review]
LGTM.

**Review by pharingee — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Comment by dr3** [comment]
Needs testing to ensure no regression in onward journeys and page loading

**Comment by PriyaKR** [comment]
Looks good to me.

**Pull request merged**